packages/e2e-tests/specs/editor/various/preview.test.js

```diff
@@ -239,13 +238,13 @@ describe( 'Preview with Custom Fields enabled', () => {
 		// Return to editor and modify the title and content.
 		await editorPage.bringToFront();
 		await editorPage.click( '.editor-post-title__input' );
-		await pressKeyWithModifier( 'primary', 'a' );
-		await editorPage.keyboard.press( 'Delete' );
-		await editorPage.keyboard.type( 'title 2' );
+		await editorPage.keyboard.press( 'End' );
+		await editorPage.keyboard.press( 'Backspace' );
+		await editorPage.keyboard.type( '2' );
 		await editorPage.keyboard.press( 'Tab' );
-		await pressKeyWithModifier( 'primary', 'a' );
-		await editorPage.keyboard.press( 'Delete' );
-		await editorPage.keyboard.type( 'content 2' );
+		await editorPage.keyboard.press( 'End' );
+		await editorPage.keyboard.press( 'Backspace' );
+		await editorPage.keyboard.type( '2' );
```

Comment:
I changed the above test because it was failing specifically on Travis and not when testing manually or running the e2e tests locally. I throttled CPU, but the tests still passed locally. I've looked into this issue for over a day, and couldn't find what's causing the problem.

I did find a reduced test case that fails:

```js
it( 'test', async () => {
  await page.keyboard.type( 'title 1' );
  await page.keyboard.press( 'Tab' );
  await page.keyboard.type( 'content 1' );
  await page.click( '.editor-post-title__input' );
  await pressKeyWithModifier( 'primary', 'a' ); // <== Remove this line and the test passes...
  await page.keyboard.type( 'title 2' );
  await page.keyboard.press( 'Tab' );
  await pressKeyWithModifier( 'primary', 'a' );
  await page.keyboard.type( 'content 2' );

  expect( await getEditedPostContent() ).toMatchSnapshot();
} );
```

So it looks like it has something to do with `Cmd+A` in the title, before doing the same in a paragraph. It's worth mentioning that we are emulating `Cmd+A`, so maybe it has to do with that and a change to the paragraph block.